src/index.js

```diff
@@ -173,8 +175,15 @@ class AvatarEditor extends React.Component {
 
     // create a canvas with the correct dimensions
     const canvas = document.createElement('canvas')
-    canvas.width = cropRect.width
-    canvas.height = cropRect.height
+
+    if (this.isVertical()) {
```

Comment:
Minor additional fix - while the canvas in the editor itself has its x and y dimensions swapped in vertical orientations, the detached canvas in here previously did not, which caused images to be squashed in non-square canvases.